Cargo.toml

```diff
@@ -42,6 +42,7 @@ url = "2.3.1"
 idna = "0.3.0"
 base64 = "0.13.1"
 num-bigint = "0.4.3"
+python3-dll-a = "0.2.7"
```

Comment:
@davidhewitt any idea why pypy is now failing, could it be this? I've no idea what else could be causing the failure.

Reply:
ignore me, it was something else.